wp-content/themes/reconasia/assets/_scss/components/_post-block.scss

```diff
@@ -1,22 +1,31 @@
 @use 'abstracts' as *;
 
 .post-block {
-  &__title {
-    color: $color-black-100;
+  margin-top: rem(40);
 
-    a {
-      &:visited {
-        // color: $color-text-gray-70;
-      }
+  &__title {
```

Comment:
Don't forget the hover effect.  The styles are at the top of the components page in a box called Post Title Link.